common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -482,6 +482,8 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
   def metricsAppTopDiskUsageCount: Int = get(METRICS_APP_TOP_DISK_USAGE_COUNT)
   def metricsAppTopDiskUsageWindowSize: Int = get(METRICS_APP_TOP_DISK_USAGE_WINDOW_SIZE)
   def metricsAppTopDiskUsageInterval: Long = get(METRICS_APP_TOP_DISK_USAGE_INTERVAL)
+  def computeUserResourceResourceConsumptionInterval: Long =
```

Comment:
repeated words here, should be computeUserResourceConsumptionInterval

Reply:
done